core/src/wsv.rs

```diff
@@ -111,18 +287,105 @@ pub struct WorldStateView {
     /// Hashes of transactions mapped onto block height where they stored
     pub transactions: HashMap<HashOf<VersionedSignedTransaction>, u64>,
     /// Buffer containing events generated during `WorldStateView::apply`. Renewed on every block commit.
+    #[serde(skip)]
     pub events_buffer: Vec<Event>,
     /// Accumulated amount of any asset that has been transacted.
+    #[serde(skip)]
     pub metric_tx_amounts: f64,
     /// Count of how many mints, transfers and burns have happened.
+    #[serde(skip)]
     pub metric_tx_amounts_counter: u64,
     /// Engine for WASM [`Runtime`](wasm::Runtime) to execute triggers.
+    #[serde(skip)]
     pub engine: wasmtime::Engine,
 
     /// Reference to Kura subsystem.
+    #[serde(skip)]
     kura: Arc<Kura>,
 }
 
+/// Context necessary for deserializing [`WorldStateView`]
+pub struct WsvSeed {
```

Comment:
Naming inconsistency: `WsvSeed` is named after what it deserializes (WSV), while `WasmSeed` is named after what it contains (wasmtime runtime).

Maybe name it `KuraSeed` for consistency?

Reply:
That's a really good catch.

Reply:
#3800